src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationService.kt

```diff
@@ -837,10 +837,26 @@ class AdjudicationService(
     return UpdateHearingResultAwardResponses(awardsCreated = createdAwards, awardsDeleted = deletedAwards)
   }
 
-  private fun deletedRemovedHearingResultAwards(
+  @Audit
```

Comment:
Very very minor, but redundant since we are only deleting so there is no audit column updated